pkg/storaged/dialog.jsx

```diff
@@ -563,6 +564,27 @@ export const TextInput = (tag, title, options) => {
     };
 };
 
+const PassInputElement = ({ tag, title, options, autoComplete, val, change, validated }) => {
+    const [show, setShow] = useState(false);
+
+    return (
+        <InputGroup>
+            <TextInputPF4 data-field={tag} data-field-type="text-input"
+                          validated={validated}
+                          disabled={options.disabled}
+                          aria-label={title}
+                          autoComplete={autoComplete}
+                          type={show ? "text" : "password"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230613-094309-5e721175-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)